### PR TITLE
Passing dilation argument to account for API change.

### PR DESCRIPTION
**Pull request by anijain2305**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

**Comment by anijain2305** [comment]
@yzhliu @shoubhik @rankyung-hong

File: topi/recipe/conv/test_conv_int8_intel.py [hunk 1]
```diff
@@ -112,8 +112,8 @@ def run_inference(data_dtype, kernel_dtype, out_dtype, im_height, im_width, in_f
 
     with tvm.target.create(TARGET_NAME):
         conv = topi.nn.conv2d_NCHWc(data, kernel, stride=hstride,
-                                    padding=hpad, layout='NCHWc',
-                                    out_layout='NCHWc', out_dtype=out_dtype)
+                                    padding=hpad, dilation=(1, 1),
```
Comment (shoubhik) [inline topi/recipe/conv/test_conv_int8_intel.py]: is dilation=(1,1) not already a default value in the `topi.nn.conv2d_NCHWc` API? if not should it be?

Comment (anijain2305) [reply topi/recipe/conv/test_conv_int8_intel.py]: This is a low-level topi operator and not the Relay operator. Relay operator has dilation default arguments which will be passed on to TOPI, if dilation was not provided in the framework graph. So, I think not having a default here is fine.

**Comment by anijain2305** [comment]
@tqchen The change is in the test case itself. Please let me know if I should add more testing to it.

**Pull request merged**